app/addons/documents/doc-editor/components.react.jsx

```diff
@@ -253,7 +277,7 @@ define([
       var iconClasses = 'icon ' + this.props.iconClass;
       return (
         <div className="panel-section">
-          <button className="panel-button upload" title={this.props.title} onClick={this.props.onClick}>
+          <button className={`panel-button ${this.props.className}`} title={this.props.title} onClick={this.props.onClick}>
```

Comment:
Maybe a default prop? I'm not sure without testing this that it doesn't output "undefined" or something in the DOM.